packages/former/src/components/FormDragContainer.vue

```diff
@@ -1,14 +1,21 @@
 <template>
-  <div class="relative flex flex-col gap-3" :class="{ 'former-drag-container min-h-24 w-full': mode === 'build' }" :data-parent-node="node?._id" :data-category="category">
+  <div
+    class="relative flex flex-col gap-3 before:content-[attr(data-drag-hint)] before:hidden before:m-auto before:text-center"
+    :class="{ 'former-drag-container min-h-24 w-full border-2 border-dashed rounded-md border-zinc-200 empty:before:block empty:bg-zinc-100': mode === 'build' }"
+    :data-parent-node="node?._id"
+    :data-category="category"
+    :data-drag-hint="texts.dragHint"
+  >
     <slot />
   </div>
 </template>
 
 <script setup lang="ts">
-import { inject } from 'vue';
+import { inject } from '~/compositions/injectProvide';
 import type { InternalSchemaNode } from '~/types';
 
 defineProps<{ node?: InternalSchemaNode; category?: string }>();
 
 const mode = inject('mode');
+const texts = inject('texts');
```

Comment:
@mariusheine why do we need to inject texts like this when it only has the dragHint there. Can we not have that string directly inside this component? or is there a future use for texts aswell?